src/commands/channel/promote.ts

```diff
@@ -58,6 +58,7 @@ export default class Promote extends SfCommand<PromoteResult> {
       char: 'p',
       summary: messages.getMessage('flags.platform.summary'),
       options: ['win', 'macos', 'deb'],
+      required: true,
```

Comment:
I suggested this to check if the error was a type issue but this flag shouldn't be required, it will break our pipelines using this command, like this one:
https://github.com/salesforcecli/github-workflows/blob/c7e9d40bd3cadc88584846a4042be49f2caa6483/.github/workflows/tarballs.yml#L53